### PR TITLE
Choose final datasets, unify EDA and prepare data for training

### PR DESCRIPTION
**Pull request by andrmdmd**

### CODE REVIEW

File: src/eda/eda.py [hunk 1]
```diff
@@ -145,9 +168,10 @@ def feature_variance_explained(dataset, n_components=None):
 
     return feature_importance.sort_values(by="Variance Explained", ascending=False)
 
+
 def train_logistic_regression(
-        dataset, regularization_type=None, c=1.0, solver='liblinear', max_iter=500
-    ):
+        dataset, regularization_type=None, c=1.0, solver='liblinear', max_iter=500, verbosity=2
+):
```
Comment (mytkom) [inline src/eda/eda.py:174]: I guess if you would modify this method to take the "LogisticRegression" object, it would work well with our implementation. (The exact `fit` and `predict` interfaces).

In the future, we can generalize this method and move it out of the EDA file. Maybe instead of printing this confusion matrix, etc., it can return some `Metrics` class object, which has the print method overridden. Hence, it calculates all metric data in the constructor and prints it in the desired format when the user wants it to.

File: src/eda/eda.py [hunk 2]
```diff
@@ -4,11 +4,13 @@
...
 import pandas as pd
 import numpy as np
+import missingno as msno
 from sklearn.manifold import TSNE
 from sklearn.linear_model import LogisticRegression
 from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
 from sklearn.decomposition import PCA
 
+
 def print_basic_info(dataset):
```
Comment (mytkom) [inline src/eda/eda.py:14]: You do not need to implement it in any way for now. But I wonder if nearly all of these media methods take `dataset` as the first parameter and do something with it. Maybe all of them should be implemented as the `DataInterface` class methods?

**Review by mytkom — APPROVED** [review]
It is neat that all these methods are reused with DataInterface in different notebooks—only small thoughts about structure.
